### PR TITLE
Build on PR

### LINKED ISSUE #1619: Upload build on PR for reviewers

### PR DESCRIPTION
**Pull request by erri120**
Resolves #1619.

Testing out CI, don't merge yet.

### CODE REVIEW

File: .github/workflows/git-builds.yaml [hunk 1]
```diff
@@ -31,7 +34,7 @@ jobs:
       # this is a bit of a hack, you can't use environment variables here
       # https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability
       ProjectFile: ${{ needs.transformInputs.outputs.ProjectFile }}
-      RetentionDays: 7
+      RetentionDays: 2
```
Comment (Al12rs) [inline .github/workflows/git-builds.yaml:37]: For daily we need 3 I think for the weekend.

**Comment by Al12rs** [comment]
For artifacts on PR, they would basically get triggered on every commit, I'm a bit hesitant for the amount of artifacts this would produce, unless we have a way to manually discard them on new commits.

**Comment by Al12rs** [comment]
Tests are failing though

**Comment by erri120** [comment]
> Tests are failing though

Tests aren't failing. No tests are being run on this PR.

**Comment by Al12rs** [comment]
Strange I could swear I was typing on the other PR?

**Pull request closed without merging**